source/whats-new.txt

```diff
@@ -32,6 +32,29 @@ Learn what's new in:
 Upcoming Breaking Changes
 -------------------------
 
+.. Relocate these items into the source/upgrade.txt page once they become breaking changes
+
+The following changes affect future versions of the driver:
+
+- The ``getStats()`` and ``isCapped()`` instance methods of the
+  ``DBCollection`` class are deprecated in v4.11. The corresponding server
+  commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats`` aggregation
+  pipeline stage to retrieve the information provided by these methods
+  instead. You can run the aggregation as shown in the following code example:
+
+  .. code-block:: java
+
+     collection.aggregate(Arrays.asList(
```

Comment:
Just checking whether you tested this, since I have not.

Reply:
Yes, I tested it. I posted valid code from which I extracted this snippet in our Slack channel when asking whether there was a more idiomatic way to get that data.